build.py

```diff
@@ -486,17 +486,16 @@ def docker_build() -> None:
 
 
 def docker_run() -> None:
-    if does_container_exist():
-        print("pgai container already exists", file=sys.stderr)
-        sys.exit(1)
+    #if does_container_exist():
```

Comment:
why is this commented out?

Reply:
should have gone in this one https://github.com/timescale/pgai/pull/71
I'll back it out